client/src/views/NewInterview.js

```diff
@@ -243,10 +268,19 @@ const InterviewPage = () => {
             </Chip>
           </div>
           <div>
+            {/* <QuestionCategoryModal
```

Comment:
If this is not necessary, remove the code.

Reply:
ok